docs/data/data-grid/overview/overview.md

```diff
@@ -7,41 +7,45 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/grid/
+<p class="description">A fast and extendable React data table and React data grid. It's a feature-rich component available with MIT or Commercial licenses.</p>
 
-The component leverages the power of React and TypeScript, to provide the best UX while manipulating an unlimited set of data. It comes with an intuitive API for real-time updates, accessibility, as well as theming and custom templates, all with blazing-fast performance.
+The component leverages the power of React and TypeScript, to provide the best UX while manipulating an unlimited set of data.
+It comes with an intuitive API for real-time updates, accessibility, as well as theming and custom templates, all with blazing-fast performance.
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
 ## Overview
 
+The Data Grid presents information in a structured format of rows and columns.
+The data is displayed in a user-friendly, quick-to-scan and interactive way, enabling users to efficiently identify patterns, edit data, and gather insights.
+
 :::info
 Visit the [installation guide](/x/react-data-grid/getting-started/#installation) to learn how to install the correct package version and dependencies.
 :::
 
-The Data Grid presents information in a structured format of rows and columns. The data is displayed in a user-friendly, quick-to-scan and interactive way, enabling users to efficiently identify patterns, edit data, and gather insights.
-
 The component comes in three different versions, one available under MIT license and two available under commercial license.
 
-## MIT version (free forever)
+## MIT license (free forever)
 
-The first version is meant as a stronger alternative to [data tables](/material-ui/react-table/#sorting-amp-selecting). It's a clean abstraction with basic features like editing, pagination, sorting and filtering single columns, and column groups.
+The MIT licensed version is a stronger alternative to [plain data tables](/material-ui/react-table/#sorting-amp-selecting).
+It's a clean abstraction with basic features like editing, pagination, sorting and filtering single columns, and column groups.
 
 ```js
 import { DataGrid } from '@mui/x-data-grid';
 ```
 
 {{"demo": "DataGridDemo.js", "defaultCodeOpen": false, "bg": "inline"}}
 
-## Commercial versions
+## Commercial licenses
 
-The commercial versions are available in the form of two plans: Pro and Premium.
+The commercially licensed versions are available in two plans: Pro and Premium.
 
 ### Pro plan [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
 
-The Pro version includes and extends the features available in the MIT version to support more complex use cases. It adds new features like advanced filtering, column pinning, column and row reordering, support for tree data, and virtualization to handle bigger datasets.
+The Pro plan includes and extends the features available in the MIT version to support more complex use cases.
```

Comment:
Are we sure we want to use the formulation "MIT version" here since your removed it above?

Reply:
Good catch—these should be "plan" as well.